src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -6268,12 +6270,10 @@ static void addTie(const Notation& notation, Score* score, Note* note, const tra
             tie->setColor(color);
         }
 
-        if (orientation == "over") {
+        if (orientation == "over" || placement == "above") {
             tie->setSlurDirection(DirectionV::UP);
-        } else if (orientation == "under") {
+        } else if (orientation == "under" || placement == "below") {
             tie->setSlurDirection(DirectionV::DOWN);
-        } else if (orientation == "auto") {
```

Comment:
why dropping the detection oa "auto"? Now in thsioe cases an error gets looged

Reply:
Because there is no value "auto", see https://www.w3.org/2021/06/musicxml40/musicxml-reference/data-types/over-under/

Reply:
Ah, I see